java/arcs/core/data/Capabilities.kt

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ *
+ * Code distributed by Google as part of this project is also subject to an additional IP rights
+ * grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+package arcs.core.data
+
+data class Capabilities(val capabilities: Set<Capability>) {
+    fun isEmpty(): Boolean = capabilities.isEmpty()
+
+    fun isSame(other: Capabilities): Boolean = capabilities.equals(other.capabilities)
```

Comment:
Can this be removed, in favor of using the `.equals` generated by the `data class` infrastructure?